tests/benchmark/s2n_pem_parse_benchmark.cc

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <benchmark/benchmark.h>
```

Comment:
Is this file part of this PR?

Reply:
This comes from google
Benchmark which is installed as part of this pr when running benchmark tests.

I haven’t completed the CI work yet, thanks for the early feedback